packages/idos-sdk-js/README.md

```diff
@@ -520,6 +520,16 @@ if (!hasProfile) window.location = "https://kyc-provider.example.com/enroll";
 const { humanId } = await idos.setSigner(CHAIN_TYPE, signer);
 ```
 
+### Discovering Human's Encryption Key
```

Comment:
Please don't pile on more to this doc. :sweat_smile: 

We need to figure out a better way to evolve how we're documenting things.